packages/app-mobile/components/side-menu-content.tsx

```diff
@@ -583,34 +581,22 @@ const SideMenuContentComponent = (props: Props) => {
 		items = items.concat(folderItems);
 	}
 
-	const isHidden = !props.sideMenuVisible;
-
 	const style = {
 		flex: 1,
 		borderRightWidth: 1,
 		borderRightColor: theme.dividerColor,
 		backgroundColor: theme.backgroundColor,
-
-		// Have the UI reflect whether the View is hidden to the screen reader.
-		// This way, there will be visual feedback if isHidden is incorrect.
-		opacity: isHidden ? 0.5 : undefined,
 	};
 
 	return (
-		<AccessibleView
-			style={style}
-
-			// Accessibility, keyboard, and touch hidden.
-			inert={isHidden}
-			refocusCounter={isHidden ? undefined : 1}
-		>
+		<View style={style}>
```

Comment:
The "hide for accessibility" and autofocus logic is now handled by `SideMenu.tsx`, so this workaround is no longer necessary.